config/mappings/test_mapping_config.py

```diff
@@ -0,0 +1,7 @@
+import json
+from jsonschema import validate
+from assessment_mapping_schema import top_level_assessment_mapping_schema as schema
+from assessment_mapping import cof_r2w2_assessment as mapping
+
+validate(instance=mapping, schema=schema)
+print("Mapping conforms with schema.")
```

Comment:
This could maybe be added to the tests that are ran by `pytest`?

Reply:
Good call, ill hold off for now as this might be moving to assessment